services/QuillLMS/app/queries/admin_diagnostic_reports/student_count_by_filter_scope_query.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module AdminDiagnosticReports
+  class StudentCountByFilterScopeQuery < ::QuillBigQuery::MaterializedViewQuery
+    class InvalidDiagnosticIdError < StandardError; end
+
+    attr_reader :timeframe_start, :timeframe_end, :school_ids, :grades, :teacher_ids, :classroom_ids, :user, :additional_aggregation, :diagnostic_id
+
+    DIAGNOSTIC_ORDER_BY_ID = [
```

Comment:
This list can be accessed from DiagnosticAggregateQuery, so we don't need to reimplement it here